services/ui-src/src/components/accordions/TemplateCardAccordion.tsx

```diff
@@ -39,7 +43,14 @@ export const DueDateDropdown = ({ verbiage }: Props) => {
               <Flex sx={sx.accordionPanelFlex}>
                 <Text sx={sx.accordionBodyTitle}>{verbiage.bodyTitle}</Text>
                 <Text sx={sx.accordionBodyText}>{verbiage.bodyText}</Text>
-                {verbiage.table && <DueDateTable verbiage={verbiage.table} />}
+                {verbiage.table && (
+                  <Table
+                    content={verbiage.table}
+                    variant="striped"
+                    lastCellsBold
+                    dataTestId="template-card-accordion-table"
+                  />
```

Comment:
Made the Table component a bit more reusable with a couple props. Idk if we'll need other tables but 🤷